### PR TITLE
infinitepower18.WSASideloader version 1.3.5

### PR DESCRIPTION
**Pull request by infinitepower18**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69354)

### CODE REVIEW

**Comment by krystofkrticka** [comment]
#68845

File: manifests/i/infinitepower18/WSASideloader/1.3.5/infinitepower18.WSASideloader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: infinitepower18.WSASideloader
+PackageVersion: 1.3.5
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/infinitepower18/WSA-Sideloader/releases/download/v1.3.5/WSASideloader_1.3.5_x64.exe
+  InstallerSha256: C888178D2E6E3A2FF674885E58969393B5A90213F75C335C877E0321CE5D928D
```
Comment (ItzLevvie) [inline manifests/i/infinitepower18/WSASideloader/1.3.5/infinitepower18.WSASideloader.installer.yaml:10]:
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/infinitepower18/WSA-Sideloader/releases/download/v1.3.5/WSASideloader_1.3.5_x64.exe
  InstallerSha256: C888178D2E6E3A2FF674885E58969393B5A90213F75C335C877E0321CE5D928D
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/infinitepower18/WSA-Sideloader/releases/download/v1.3.5/WSASideloader_1.3.5_x64.exe
  InstallerSha256: C888178D2E6E3A2FF674885E58969393B5A90213F75C335C877E0321CE5D928D
  Scope: machine
  InstallerSwitches:
    Custom: /ALLUSERS
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/i/infinitepower18/WSASideloader/1.3.5/infinitepower18.WSASideloader.locale.en.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: infinitepower18.WSASideloader
+PackageVersion: 1.3.5
+PackageLocale: en
+Publisher: infinitepower18
+PackageName: WSA Sideloader
+License: GPL v3
```
Comment (ItzLevvie) [inline manifests/i/infinitepower18/WSASideloader/1.3.5/infinitepower18.WSASideloader.locale.en.yaml:9]:
```suggestion
License: GPL-3.0-only
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**